tests/python/gaia-ui-tests/gaiatest/apps/camera/app.py

```diff
@@ -94,11 +88,12 @@ def tap_select_button(self):
         select = self.marionette.find_element(*self._select_button_locator)
         Wait(self.marionette).until(expected.element_enabled(select))
 
-        try:
-            select.tap()
-        except (FrameSendFailureError, NoSuchWindowException):
-            # The frame may close for Marionette but that's expected so we can continue - Bug 1065933
-            pass
+        # Workaround for bug 1109213, where tapping on the button inside the app itself
+        # makes Marionette spew out NoSuchWindowException errors
+        x = select.rect['x'] + select.rect['width']//2
+        y = select.rect['y'] + select.rect['height']//2
+        self.marionette.switch_to_frame()
+        self.marionette.find_element(*self._screen_locator).tap(x=x, y=y)
```

Comment:
just curious, if this tap method is from marionette.py, then can't we just do .tap(x,y) since they are the only parameters?